Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1072,14 +1072,17 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Geom' : 'Extended2026D64',                   # N.B.: Geometry with square 50x50 um2 pixels in the Inner Tracker.
         'HLTmenu': '@fake2',
         'GT' : 'auto:phase2_realistic_T22',
+        'ProcessModifier': 'phase2_PixelCPEGeneric',  # This swaps template reco CPE for generic reco CPE
+        'Custom' : 'RecoLocalTracker/SiPixelRecHits/CustomPixelCPEGeneric.customizeForSquarePixels',
         'Era' : 'Phase2C11',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
     },
     '2026D65' : {
         'Geom' : 'Extended2026D65',                   # N.B.: Geometry with 3D pixels in the Inner Tracker.
         'HLTmenu': '@fake2',
         'GT' : 'auto:phase2_realistic_T23',           # This symbolic GT has no pixel template / GenError informations.
-        'ProcessModifier': 'phase2_PixelCPEGeneric',  # This modifier removes all need for IT template information. DO NOT USE for standard planar sensors.
+        'ProcessModifier': 'phase2_PixelCPEGeneric',  # This swaps template reco CPE for generic reco CPE
+        'Custom' : 'RecoLocalTracker/SiPixelRecHits/CustomPixelCPEGeneric.customizeFor3DPixels',
```

Comment:
do we expect specifications for 3D or square pixels in any other places?
Normally, non-temporary modifications should be done via ProcessModifier

Reply:
I am not expecting other specifications at the moment, but I cannot fully excluded them.
By construction this is a temporary modification as the final geometry should crystalize by the end of the current year.
By the way, we need to run already a `ProcessModifier` to swap template reco with generic reco, are we allowed to pile-up more than one `ProcessModifier`? 
If that's case what's the syntax?

Reply:
For two modifiers `x` and `y`, you can use e.g. `(x & y).toModify(...)`.

Reply:
how do I pass them both in the upgrade matrix?

Reply:
Ah, you can pass multiple modifiers separated by commas in a single string. It will be parsed correctly by cmsDriver.

Reply:
@slava77 I prepared (and validated) an alternative version of this at https://github.com/cms-sw/cmssw/compare/master...mmusich:fix_Phase2_ITReco_11_3_X_v2 based on `ProcessModifier`s in lieu of customization functions. 
If there is a strong push for it, I'll push it here. Let me know.

Reply:
> @slava77 I prepared (and validated) an alternative version of this at [master...mmusich:fix_Phase2_ITReco_11_3_X_v2](https://github.com/cms-sw/cmssw/compare/master...mmusich:fix_Phase2_ITReco_11_3_X_v2) based on `ProcessModifier`s in lieu of customization functions.
> If there is a strong push for it, I'll push it here. Let me know.

Thank you.
I suppose, the customization methods are not needed in this case.

BTW, shouldn't the `phase2_3DPixels` and `phase2_SquarePixels` be made a part of the corresponding Phase2 eras? The procModifiers are more like switches, somewhat agnostic of the detector specifics, while the eras are more specific to the detector design. We still do not have any `Phase2*Tnn` eras though.

Reply:
Indeed, so far we have mainly had subdetector-version-dependent Eras for HGCal changes (though the proliferation has already started to increase, with some for Muon and MTD introduced recently). If the changes are really specific to certain Tracker geometries, then we should probably introduce specific Eras, though this does complicate the overall bookkeeping. (I also encourage updating the "era" fields in https://github.com/cms-sw/cmssw/blob/master/Configuration/Geometry/python/dict2026Geometry.py accordingly, though these are not automatically enforce, but rather just serve as a reminder whenever generating a new geometry.)

Reply:
mhm, wouldn't we then be coupled to some other detector configuration? From which phase2 era should one start?  Right now square pixels are bound to D64 and 3D pixels to D66 but I understood from @srimanob that these might be migrated later to other detector combinations. Wouldn't then be a maintenance overhead to keep updating the eras? I'd rather not do that.

Reply:
Yes, I think we'd ultimately need a ModifierChain for each subdetector version and then an Era for each full detector version. If the tracker group would prefer just to attach some ProcessModifiers to the relevant workflows for now, that's fine with me. (However, be prepared for someone to copy a config from someone else, lose track of the ProcessModifiers, and then complain that it doesn't work...)

Reply:
Just to be sure I understood correctly, is this  https://github.com/cms-sw/cmssw/compare/master...mmusich:fix_Phase2_ITReco_11_3_X_v3 what you had in mind?

Reply:
Mostly... in general I'd prefer the Era names to use the subdetector identifiers from https://github.com/cms-sw/cmssw/tree/master/Configuration/Geometry (which never change) rather than names of specific features, whose association with a subdetector version might not be obvious. But that's not a strict requirement.

Reply:
so I would have to call the eras something like `Era_Phase2C11_T22_cff.py ` and `Era_Phase2C11_T23_cff.py `?

Reply:
I think the middle underscore is unnecessary; I would just use `Phase2C11T22` etc.

Reply:
Done as suggested in the last push